### ISSUE TITLE
[Project 3] Planning Meeting (11/09/21 )

### ISSUE THREAD

**Issue by henrylao** [akailany/Pattern-Segmentation-in-Aerial-Agricultural-Images#7]
# Project 3 Specifications
* __DUE DATE: Nov 30__
* Guidelines: http://borg.csueastbay.edu/~grewe/CS663/EvalGuidelines/Project3_Guidelines.html

# Development Resources:
- MSCG-Net: https://github.com/samleoqh/MSCG-Net/tree/master
- Dataset: https://www.agriculture-vision.com/agriculture-vision-2021/dataset-2021

# Issues
__EVERYONE__
- [ ] Read MSCG-Net: https://arxiv.org/pdf/2004.10327.pdf

__Backend__
- MSCG-Net: complaining about missing module `lib.*` that exists within the in-house API 
- Train + Checkpoints
- Tensorboard + relevant stats (train, valid loss + accuracy)
   - Other metrics...? 
- Convert to Torch Mobile (Torchscript to Mobile)

__Frontend__
- Android activities GUI layout implementation
- GUI diagram

# Deliverables
- Weekly Progress Report Submissions
__NOTE__: documentation accountability is based on area of work ie frontend does frontend docs, backend does backend docs
  - [x] Sprint 1: 11/12/21 __DUE 11PM__
         - Frontend: User interface, frontend layout documentation, and implementation
          - Backend: Read the [paper](https://arxiv.org/pdf/2004.10327.pdf), testing + fixing existing PyTorch codebase
  - [ ] Sprint 2: 11/19/21 __DUE 11PM__
         - Frontend:  
         - Backend:
  - [ ] Sprint 3: 11/26/21 __DUE 11PM__ **Q! Meeting during TY-Giving?**
          - Documentation
         - Frontend:  
         - Backend: Fine-tuning if necessary 

- [ ] Android Source Code (zipped) **Q! ml code?**
- [ ] PDF Document containing the following
- [ ] __Documentation__
> Instructions for me to download and run your code. YOU NEED to show me screenshots of you doing this from your uploaded blackboard code.....this forces you to make sure that I can run your code. You MUST have the following screenshots AND give a description on what to do: screenshot 1.1 = screenshot of your files uploaded to Project 1 turn in a folder on blackboard
- [ ] Code Description 
>  A describing how code is structured and the state of how it works. Give a description for each filename listed. 
- [ ] Testing
> Section 3 Testing:
 here you give screenshots of you running the various stages of the program as detailed here: 
-  [ ] YouTube Demo
    - what is working
    - what is not working 
    - how well it works
